library/Stratosphere/Types.hs

```diff
@@ -53,13 +53,13 @@ data AuthorizationType
   deriving (Show, Read, Eq, Generic, ToJSON)
 
 data HttpMethod
-  = GET
-  | POST
-  | PUT
-  | HEAD
+  = ANY
```

Comment:
This first commit exists so the later addition can place the `PATCH` to a location that can be explained. Before I'd have had to put it "somewhere" in an unordered list.

I'm fine to drop this commit if controversial.